app/views/places/index.html.erb

```diff
@@ -1,30 +1,28 @@
-<h1>Places</h1>
+<div class="jumbotron jumbotron-fluid">
+  <div class="container">
+    <h1 class="display-4">Hello, world!</h1>
+    <p class="lead">This is a simple hero unit, a simple jumbotron-style component for calling extra attention to featured content or information.</p>
+    <hr class="my-4">
+    <p>It uses utility classes for typography and spacing to space content out within the larger container.</p>
+    <a class="btn btn-primary btn-lg" href="/events" role="button"><%= t('.showmeevents') %></a>
+  </div>
+</div>
 
-<table>
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Address</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
+<div class=container>
```

Comment:
Are you sure you resolve all conflicts in the right way? We talk about this missing `""` for class name.

Reply:
Missing `""` around `container` class

Reply:
kurcze... to jest branch translations który ja widze:
<div class=container>
      <div class="row">
        <% @places.each do |place| %>
          <div class="col-md-4">
            <h2><%= place.name %></h2>
            <p><%= place.address %></p>
            <p><%= link_to t('.show'), place_path(place), class: "btn btn-secondary btn-xs" %> </p>
            <p><%= link_to t('.edit'), edit_place_path(place) %> <%= link_to t('.destroy'), place, method: :delete, data: { confirm: 'Are you sure?' } %></p>
          </div>
        <% end %>
    <%= link_to t('.newplace'), new_place_path %>
    </div>
</div>  <%#container%>